src/views/Organisations/Organisation.vue

```diff
@@ -631,6 +672,17 @@ export default {
       );
       await this.getCountries();
       this.loading = false;
+    },
+    async deleteOrganisation(del) {
+      if (del) {
+        let data = await restClient.deleteOrganisation(this.organisation.id, this.user().credentials.token);
+        if (!data.error) {
+          // Deploy to organisations page.
+          const url = new URL(process.env.VUE_APP_HOSTNAME + '/organisations');
```

Comment:
I tried the $router.push you suggested. But, it doesn't reload the data and so it's possible to delete an organisation and be directed back to the list only to find the organisation still listed there. I suspect that a user would complain sooner or later so using window.location.href solves this. 
Is it just the double slash that's causing issues? I could fix that.

Reply:
I have fixed the issue using ```window.location.pathname = '/organisations'```.
I have to do force push in this case. I am not sure but the I was getting ```git pull ``` before ```git push``` and the ```git pull``` was somehow not being successfull.

Reply:
Strange that it should not be successful. Please let me know if it happens again as I'd like to have  a look in case anything stands out as a potential cause.